backend/routes/colors.go

```diff
@@ -40,6 +40,7 @@ func InitColors(w http.ResponseWriter, r *http.Request) {
 }
 
 func GetAllColors(w http.ResponseWriter, r *http.Request) {
+	SetupHeaders(w)
```

Comment:
There is no need to add `SetupHeaders` to any of the routes, because it is already handled in the the `WriteErrorJson` / `WriteResultJson` / ...